src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -353,7 +353,7 @@ private OutputAlleleSubset calculateOutputAlleleSubset(final AFCalculationResult
             if (alternativeAllele.isReference()) {
                 continue;
             }
-            final boolean isPlausible = afCalculationResult.isPolymorphicPhredScaledQual(alternativeAllele, configuration.genotypeArgs.STANDARD_CONFIDENCE_FOR_EMITTING);
+            final boolean isPlausible = afCalculationResult.isPolymorphicPhredScaledQual(alternativeAllele, configuration.genotypeArgs.STANDARD_CONFIDENCE_FOR_CALLING/3);
```

Comment:
Either make the 3 a named constant explaining what it is or make a named constant with the result of the division.